include/zephyr/drivers/video.h

```diff
@@ -33,6 +34,27 @@
 extern "C" {
 #endif
 
+/**
+ * @brief Print formatter to use in printf() style format strings.
+ *
+ * This prints a video format in the form of "{width}x{height} {fourcc}" format
```

Comment:
From macro below, should be "{fourcc} {width}x{height}" ?

Reply:
You are right, I changed it after seeing how `v4l2-ctl` and the examples show the formats but forgot to update thedoc. I will adjust it along with other doc fixups.
Thank you for pointing it!